MatterControl.Printing/PrinterCommunication/PrinterConnection.cs

```diff
@@ -1121,12 +1125,14 @@ public void Connect()
 On Linux, MatterControl requires a serial helper library in order to use certain baud rates. It is possible that this component is missing or not installed properly. ";
 											}
 											StyledMessageBox.ShowMessageBox(message, "Unsupported Baud Rate".Localize(), useMarkdown: true);
-										});
+										}); */
 									}
 									catch (IOException e)
 									{
-										TerminalLog.WriteLine("Exception:" + e.Message);
+										this.LogError("Exception:" + e.Message);
 										OnConnectionFailed(ConnectionFailure.IOException);
+										// TODO: Reimplement exception specific error messages
+										/*
```

Comment:
Create issue

Reply:
Issue created - MatterHackers/MCCentral#5766
Replace PrinterConnection MatterControl error messages with error and specific error codes